client/.gitignore

```diff
@@ -2,4 +2,3 @@ node_modules
 /.cache
 /build
 /public/build
-.env
```

Comment:
los archivos `.env` deben estar en el `.gitignore` porque no queremos exponer las claves por temas de seguridad.
Lo que se suele subir es un archivo `.env.template` que es como un placeholder de las variables. Luego, de forma privada, se comparten estas variables por otros medios con el resto del equipo.

puedes leer:
https://www.getfishtank.com/blog/best-practices-for-committing-env-files-to-version-control
<img width="671" alt="image" src="https://github.com/T00lio/quiz/assets/11441487/f2a9df6b-5ed4-4f3e-9586-8d078eab36c6">